application/src/Mvc/MvcListeners.php

```diff
@@ -144,7 +144,7 @@ public function bootstrapLocale(MvcEvent $event)
      * Redirect all requests to install route if Omeka is not installed.
      *
      * @param MvcEvent $event
-     * @return Zend\Http\PhpEnvironment\Response
+     * @return \Zend\Stdlib\ResponseInterface
```

Comment:
The responses being used here are actually the HTTP type indicated, and there's quite a bit of their functionality only available in those more specific subclasses, so we probably don't want to genericize this down to just the ResponseInterface.

Reply:
Ok. Fixed in next commit.